sdk/storage/azure-storage-files-datalake/cgmanifest.json

```diff
@@ -10,16 +10,6 @@
       },
       "DevelopmentDependency": true
     },
-    {
-      "Component": {
-        "Type": "git",
-        "git": {
-          "RepositoryUrl": "https://github.com/edlund/amalgamate",
```

Comment:
Why did we have a dependency on this in the first place?

@vhvb1989 looks like you added it here. Does nlohmann indirectly need it:
https://github.com/Azure/azure-sdk-for-cpp/pull/839/commits/6c24921c429b58ceea4d4d42054604b33b891d8b

Reply:
I added based on the current Notice.txt provided by storage.